### DISCUSSION TITLE
Interface Channel Fields Value

### DISCUSSION THREAD

**Discussion by moetezz** [netbox-community/netbox#15096] [Ideas]
I am proposing to add the value of "Auto" to the `Channel Width` and `Channel Frequency` Fields in the Interface Data Model.
Also proposing to Add a `Transmit Power (dBm)` Field in the Interface to have numeric values or "Auto".
Or is there a way to infer the `Auto` value from None when doing an export_template or some other way?

This would be helpful as sometimes, Operations want those values to be dynamically set rather than fixed, so this info would be transmitted to the installation team via these NetBox fields, and having this ambiguity might create problems in the future.

Would appreciate any ideas

**Comment by candlerb** [comment]
> is there a way to infer the Auto value from None when doing an export_template

Possibly `{{ obj.rf_channel | default("Auto") }}`

However if the default value is empty string rather than None, you might need to use an "if" block instead. [jinja2 docs](https://jinja.palletsprojects.com/en/3.1.x/templates/).

Having said that, having an explicit "Auto" value for channel sounds reasonable to me.

**Reply by moetezz** [reply]
Thanks for the suggestion, I think this could work. for these fields the default value can't be an empty string as they are of type numeric. So this might solve it.
